### ISSUE TITLE
Mystical class cast errors

### ISSUE THREAD

**Issue by Juuxel** [CottonMC/ClientCommands#7]
> `[16:05:21] [Server thread/INFO]: Player11 lost connection: Internal Exception: java.lang.ClassCastException: net.minecraft.server.network.ServerPlayNetworkHandler cannot be cast to net.minecraft.network.listener.ClientPlayPacketListener`

Apparently vanilla swallows the stacktrace.

**Comment by ChloeDawn** [comment]
Any update on this?

**Comment by Juuxel** [comment]
Not really, since a) I can't reproduce it consistently and b) I don't have a stacktrace.
